pkg/operator/observe_config.go

```diff
@@ -86,6 +88,45 @@ func (c ConfigObserver) sync() error {
 		unstructured.SetNestedStringSlice(observedConfig, etcdURLs, "storageConfig", "urls")
 	}
 
+	// observe configuration from cluster-config-v1
+
+	// get cluster config
+	clusterConfig, err := c.kubeClient.CoreV1().ConfigMaps("kube-system").Get("cluster-config-v1", metav1.GetOptions{})
+	if err != nil {
```

Comment:
only return the error if it is something other than NotFound.  Otherwise you will cause all observed config updates to be skipped on missing input.  I doubt this will be a final holding location, but this code here would effectively pin it there forever.